docs/docs/Get-Started/get-started-installation.md

```diff
@@ -3,170 +3,223 @@ title: Install Langflow
+## Install and run Langflow OSS
 
-### Prerequisites
+Before you install and run Langflow OSS, be sure you have the following items.
 
-- [Python 3.10 to 3.13](https://www.python.org/downloads/release/python-3100/) installed
-- [uv](https://docs.astral.sh/uv/getting-started/installation/), [pip](https://pypi.org/project/pip/), or [pipx](https://pipx.pypa.io/stable/installation/) installed
-- Before installing Langflow, we recommend creating a virtual environment to isolate your Python dependencies with [uv](https://docs.astral.sh/uv/pip/environments), [venv](https://docs.python.org/3/library/venv.html), or [conda](https://anaconda.org/anaconda/conda)
+- [Python 3.10 to 3.13](https://www.python.org/downloads/release/python-3100/)
+- [uv](https://docs.astral.sh/uv/getting-started/installation/) or [pip](https://pypi.org/project/pip/)
+- A virtual environment created with [uv](https://docs.astral.sh/uv/pip/environments) or [venv](https://docs.python.org/3/library/venv.html)
 
-### Install Langflow with pip or pipx
+Install and run Langflow OSS with [uv (recommended)](https://docs.astral.sh/uv/getting-started/installation/) or [pip](https://pypi.org/project/pip/).
 
-Install Langflow with uv:
+1. To install Langflow, use one of the following commands:
+
+<Tabs>
+<TabItem value="uv" label="uv" default>
 
 ```bash
 uv pip install langflow
 ```
 
-Install Langflow with pip:
+</TabItem>
+<TabItem value="pip" label="pip">
 
 ```bash
-python -m pip install langflow
+pip install langflow
 ```
 
-Install Langflow with pipx using the Python 3.10 executable:
-
-```bash
-pipx install langflow --python python3.10
-```
+</TabItem>
+</Tabs>
 
-## Run Langflow
+2. To run Langflow, use one of the following commands:
 
-1. To run Langflow with uv, enter the following command.
+<Tabs>
+    <TabItem value="uv" label="uv">
 
 ```bash
 uv run langflow run
 ```
 
-2. To run Langflow with pip, enter the following command.
+</TabItem>
+<TabItem value="Python" label="Python">
 
 ```bash
 python -m langflow run
 ```
 
-3. Confirm that a local Langflow instance starts by visiting `http://127.0.0.1:7860` in a Chromium-based browser.
+</TabItem>
+</Tabs>
+
+3. To confirm that a local Langflow instance starts, go to the default Langflow URL at `http://127.0.0.1:7860`.
+
+After confirming that Langflow is running, create your first flow with the [Quickstart](/get-started-quickstart).
 
-Now that Langflow is running, follow the [Quickstart](/get-started-quickstart) to create your first flow.
+### Manage Langflow OSS versions
 
-## Manage Langflow versions
+To upgrade Langflow to the latest version, use one of the following commands:
 
-To upgrade Langflow to the latest version with uv, use the uv pip upgrade command.
+<Tabs>
+<TabItem value="uv" label="uv" default>
 
 ```bash
 uv pip install langflow -U
 ```
 
-To upgrade Langflow to the latest version, use the pip upgrade command.
+</TabItem>
+<TabItem value="pip" label="pip">
```

Comment:
Is there a way to persist the tabs? In other words, if I select the pip tab in one place on this page, the pip tab displays by default in other places on this page. We have the ability to do this in Asciidoc, but I'm not sure if Docusaurus supports it.

Reply:
We can do that with group ID, I'll followon with another PR for it